packages/react-core/src/core/providers/thirdweb-wallet-provider.tsx

```diff
@@ -123,7 +126,18 @@ export function ThirdwebWalletProvider(
   }, [activeWallet, props.chains]);
 
   const handleWalletConnect = useCallback(
-    async (wallet: InstanceType<SupportedWallet>) => {
+    async (wallet: InstanceType<SupportedWallet>, params?: ConnectParams<Record<string, any>>) => {
+      const connectParams = params || walletParams;
+
+      await coordinatorStorage.setItem(
+        "lastConnectedParams",
+        JSON.stringify(connectParams),
+      );
+      await coordinatorStorage.setItem(
+        "lastConnectedWallet",
+        wallet.walletId,
+      );
```

Comment:
since these are *always* linked I think, can we just store them in the same record?